internal/cli/root.go

```diff
@@ -20,14 +19,6 @@ import (
 
 const rootShort = "Supercharge your development workflow."
 
-// authCfg defines the configurable auth context the cli will run in.
-var authCfg struct {
```

Comment:
Moved this on the Authenticator instead and abstracted it under the auth.New() call.